<html>
<head>
  <meta name="generator" content="Hands & Brain">
  <title>smBsm integrated system - DMMPMM</title>
  <link rel="stylesheet" href="dmmpmm.css" type="text/css">
</head>
<body>
  <h1>Drosophila Melanogaster Major Position Matrix Motifs (DMMPMM) <i>MAY-26-2009</i></h1>
  <img src="drosophila.png" align="left" style="margin: 0; padding: 0; padding-left: 5px;"/>
  <p>This site contains supplementary materials for the <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btp434v1">paper</a> 'Motif discovery and motif finding from genome-mapped DNase footprint data'.<br/>
  All genome-mapped data uses Drosophila melanogaster v.4 Apr. 2004 (UCSC dm2) genome assembly (that is called dmel40). All coordinates are one-based. <br/>
  Sequence logos are generated by pmflogo tool. It uses green grid for DIC-based scaling (in case PCM exists) and red grid for weblogo-like classic information content scaling (for PPM-only cases).<br/>
  Check bottom of the page for the 'downloads' section. This is static page, so if you want to see the latest version do not be afraid of hitting F5 (refresh) in your browser.
  Some specific miscellaneous details can be found <a href="addon.html">here</a>.
  </p>
  <br/>
  
  <h1 style="color: red; background: pink; border-bottom: orange 3px solid;"><b><a class="simple" href="http://autosome.ru/iDMMPMM/">[UPDATE]</a></b> improved and integrated Drosophila Melanogaster Major Position Matrix Motifs (iDMMPMM)</h1>
  <p>Newest motifs constructed by integrating different experimental data are available <a href="http://autosome.ru/iDMMPMM/">here</a> 
  according to the <a class="simple" href="http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1134/S0006350909060013">[Kulakovskiy, I.V. and Makeev, V.J. (2009) Discovery of DNA motifs recognized by transcription factors through integration of different experimental sources. Biophysics, 54(6), 667-674]</a>.</p>
  <br/>
  <p><small>Please, use <a href="http://autosome.ru/DMMPMM/">blocks</a> for navigation. Within tables factor IDs lead to the detailed data (this is not so only for features and similarity tables).</small></p>
  
  <h2 class="link"><a href="DMMPMM_18_MAR_RELEASE/comparison-DMMPMM.html">Different motif models comparison</a></h2>
  <p>There you can see comparison of motif models came from different sources. Click corresponding motif ID to see PWM ROC curves and the motif similarity table. 
  'Winner' is the motif with better sensitivity in most of the cases in range of 10-90% of total footprints count. <br/>
  </p>
  
  <p class="small">
  Color codes: <span class="yellow">yellow</span> wins in some cases, <span class="green">green</span> wins in most of the cases.
  </p>
  <br/>
  
  <h2 class="link"><a href="DMMPMM_18_MAR_CTRL/comparison-DMMPMM_ctrl.html">Control motif models comparison</a></h2>
  <p>There you can see comparison of motif models by Bigfoot and SeSiMCMC with fixed length from Pollard motif set. Click corresponding motif ID to see PWM ROC curves and the motif similarity table. 
  'Winner' is the motif with better sensitivity in most of the cases in range of 10-90% of total footprints count. <br/>
  </p>
  <br/>
  
  <h2 class="link"><a href="DMMPMM_18_MAR_RELEASE/list-BE_DMEL-footprint-Bigfoot.html">Bigfoot motifs (full list)</a></h2>
  <p> There you can see the whole list of motifs built by Bigfoot. 'ready' xml contains the motif built with the dmel40-background distribution and pseudocount taken as 1.
  'map' xml corresponds to site positioning within footprints. 'sites' xml corresponds to genome-mapped site positioning. 
  <br/>
  </p>
  <p class="small">You can check motifs with different lengths by clicking on the corresponding motif ID. 
  You can open a big motif logo by clicking on the small one.</p>  
  <br/>
  
  <h2>Motif features</h2>
  <table style="border: none; width: 99%; padding: 0;"><tr>
  <th style="font-size: 140%; padding-right: 5px; padding-left: 10px;">
  <span style="font-size: 70%">
  <p><a href="DMMPMM_18_MAR_RELEASE/features-BE_DMEL-footprint-Bigfoot.html">Bigfoot</a></p>
  <p><a href="DMMPMM_18_MAR_RELEASE/features-BE_DMEL-footprint-Down.html">Down</a>&nbsp;<small style="font-size: 65%;"><a href="http://bergmanlab.smith.man.ac.uk/wp-content/uploads/2010/02/SupplementalFile2.xms">&lt;link&gt;</a></small></p>
  <p><a href="DMMPMM_18_MAR_RELEASE/features-BE_DMEL-footprint-Pollard.html">Pollard</a>&nbsp;<small style="font-size: 65%;"><a href="http://www.danielpollard.com/matrices.html">&lt;link&gt;</a></small></p>
  <p><a href="DMMPMM_18_MAR_RELEASE/features-BE_DMEL-footprint-SeSiMCMC.html">SeSiMCMC</a>&nbsp;<small style="font-size: 65%;"><a href="http://favorov.bioinfolab.net/SeSiMCMC/">&lt;link&gt;</a></small></p>
  <p><a href="DMMPMM_18_MAR_RELEASE/features-NO_SOUR-curated-Bergman.html">Bergman</a>&nbsp;<small style="font-size: 65%;"><a href="http://bergmanlab.smith.man.ac.uk/?page_id=274">&lt;link&gt;</a></small></p>
  <p><a href="DMMPMM_18_MAR_RELEASE/features-NO_SOUR-curated-Papatsenko.html">Papatsenko</a>&nbsp;<small style="font-size: 65%;"><a href="http://flydev.berkeley.edu/cgi-bin/Annotation/index.htm#30_motifs">&lt;link&gt;</a></small></p>
  <p><a href="DMMPMM_18_MAR_RELEASE/features-NO_SOUR-specific-Noyes.html">Noyes</a>&nbsp;<small style="font-size: 65%;"><a href="http://labs.umassmed.edu/WolfeLab/data_files/Noyes_AP_factors_raw.xls">&lt;link&gt;</a></small></p>
  <p><a href="DMMPMM_18_MAR_RELEASE/features-NO_SOUR-specific-Noyes_hd.html">Noyes_hd</a>&nbsp;<small style="font-size: 65%;"><a href="http://labs.umassmed.edu/WolfeLab/data_files/84_HD_aligned_sites_omega.txt">&lt;link&gt;</a></small></p>
  </span>
  </td><td style="font-size: 80%; width: 100%; padding: 0; margin: 0;">
  <p> Each link corresponds to the motif features table for different motif set. By 'site' we mean the best PWM hit within footprint (with or without flanking sequences depending on concrete case). 
  Column descriptions: <br/>
  <b>average, the best, the worst score</b> are defined over the whole set of possible words. For motifs longer than 10 we calculate these values over the set of 4<sup>10</sup> randomly selected words
  (with uniform background distribution).</p>
  <p class="colored"><b>average+sigma,+2sigma,+3sigma</b> corresponds to the confident intervals defined by the mean and standard deviation for overall score distribution (if it's taken as Normal distribution).<br/>
  <b>sequence count</b> is the total footprint count for selected factor, <span class="green">green</span> if more than 16, <span class="yellow">yellow</span> if more than 8.<br/>
  </p>
  <p>We define site absence threshold as average+sigma. There are some cases when there are footprints containing no hits scoring better than average+sigma. In this case we can check additional columns.<br/>
  <b>sequences w/o score</b> is the number of sequences without a 'ready' PWM hit better than corresponding value
  (so <b>sequences w/o score > average+sigma</b> corresponds to the number of sequences without site).<br/></p>
  
  <p class="colored">
  <b>worst site, best site</b> is the worst and best score in the set of sites (considering the site as the best PWM hit in the footprint).</br>
  <b>flank length</b> is the flank length (selected equal to motif length) used for possible motif appearance if there were some footprints without site.<br/>
  <b>% w/o sites</b> is the percent of sequences containing no hits scoring better than site absence threshold (<span class="red">red</span> if > 10%).<br/>
  <b>sites appearing</b> column corresponds to the percent of appeared sites (<span class="green">green</span> if > 10%) after adding flanking sequences to initial footprints (site appearance threshold was set to <b>average+2sigma</b>).<br/>
  <b>%weak sites</b> corresponds to percent of sequences with the best site within [average+sigma, average+3sigma] interval (for footprints without any flanking sequences).<br/></p>
  
  <p><b>weak words in the 'ready' motif</b> corresponds to the number of sites in the alignment scoring lower than threshold for the 'ready' motif (available for Bigfoot and Noyes, uses word-list from the 'ready' motif file).</p>
  </td></tr></table>
  
  <p>It should be noted, that for each motif set we pick only motifs presented in our main Bigfoot footprint-made collection. If you are
  interested in the whole motif set you can download it in the 'downloads' section or follow concrete link to access original data. </p>
  
  <small><p>One can not be sure that links sending outside of this site are working at this moment. All links were checked at 01-march-2009.</p></small>
  <br/
  </p>
  
  <h2 class="link"><a href="DMMPMM_18_MAR_RELEASE/similarity-BE_DMEL-footprint-Bigfoot.html">Bigfoot motif similarity</a></h2>
  <p> This table corresponds to <a href="http://favorov.imb.ac.ru/ahokocc/">AhoPro</a>-based similarities of Bigfoot motifs for different factors. </p>
  <p class="small">
  Color codes for similarity coefficient scale: <span class="red">...<b>0.4</b></span><span class="orange">...<b>0.3</b></span><span class="yellow">...<b>0.2</b></span><span class="green">...<b>0.1</b></span><span class="blue">...<b>0.05</b></span><span class="violet">...<b>0.01</b></span>
  </p>
  <br/>
  
  <h2>Funding</h2>
  <p>The work was supported by Russian Fund of Basic Research projects [07-04-01623 and 07-04-01584]; INTAS Project [05-1000008-8028]; Russian Federation Agency in Science and Innovation State Contract [02.531.11.9003]; 
  Russian Academy of Sciences Program in Molecular and Cellular Biology, Project #10; and French INRIA &Eacute;quipe associ&eacute; MIGEC.</p>
  <br/>
  
  <h2>Downloads</h2>
  <p><small>DMMPMM and Ytilib source archives include minimal installation and usage instructions.</small></p>
  <p><small><i>*_motif</i> files correspond to the initial motifs (without threshold estimation), <i>*_ready</i> files correspond to the motifs (threshold included) normalized over the DMEL40 background. Sites within footprints are mapped using Bigfoot motif collection. </small></p>
  <table style="font-size: 120%">
  <tr><td>Footprint-mapped motifs and motif collections (small-BiSMark and plain text format): </td><td><a href="http://autosome.ru/DMMPMM/pack/dmmpmm_motifs.zip">dmmpmm_motifs.zip</a></td></tr>
  <tr><td>Modified DMMPMM scripts used for control motif creation and comparison: </td><td><a href="http://autosome.ru/DMMPMM/pack/dmmpmm_ctrl.zip">dmmpmm_ctrl.zip</a></td></tr>
  <tr><td>Actual DMMPMM release (including DMMPMM source and motif collections): </td><td><a href="http://autosome.ru/DMMPMM/pack/dmmpmm_release.zip">dmmpmm_release.zip</a></td></tr>
  <tr><td>Ytilib ruby library and some useful tools (including Bigfoot software): </td><td><a href="http://autosome.ru/DMMPMM/pack/ruby_ytilib.zip">ruby_ytilib.zip</a></td></tr>
  <tr><td>Bismark-adapted MySQL database dump: </td><td><a href="http://autosome.ru/DMMPMM/pack/mysql_bismark.zip">mysql_bismark.zip</a></td></tr>
  <tr><td>small-BiSMark <a href="http://autosome.ru/smallBiSMark/smallbismark.dtd">DTD</a>, samples and details: </td><td><a href="http://autosome.ru/DMMPMM/pack/bismark.zip">bismark.zip</a></td></tr>
  </table>
  <br/>
  
  <h2>Updates</h2>
  <small style="font-size: 70%;">
  <p>February 2011 - Updated link to the Bergman Lab motifs.</p>
  <p>Novermber 10 2009 - Updated link to the Bergman Lab motifs.</p>
  <p>May 26 2009 - Fixed PMW comparison graph titles; some other corrections.</p>
  <p>March 18 2009 - Renewed motifs sets. Separated control motif sets comparison.</p>
  <p>March 17 2009 - Renewed Bigfoot_ctrl set.</p>
  <p>March 11 2009 - Bigfoot_ctrl set added to the main comparison; renewed downloads section.</p>
  <p>March 03 2009 - SeSiMCMC set renewed.</p>
  <p>March 02 2009 - Fixed incorrect coloring of the main comparison table; Noyes homeodomain data added to the main comparison.</p>
  <p>March 01 2009 - Noyes data added to the main comparison.</p>
  <p>January 01 2009 - Initial public release.</p>
  </small>
  <br/>
  
  <small style="margin: 7px;">Please, send all questions regarding this site and software to <a href="#">ivan-dot-kulakovskiy-wow-gmail-dot-com</a>.</small><br/>
  <small style="margin: 7px;">Fruit fly logo is kindly presented by Irina Eliseeva.</small>

<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
var pageTracker = _gat._getTracker("UA-6896924-1");
pageTracker._initData();
pageTracker._trackPageview();
</script>

</body>
</html>
