<html>
<head>
<meta http-equiv="Content-Type" content="text/html">
<title>Yet Another Digging For DNA Motifs Gibbs Sampler (SeSiMCMC) output.</title>
</head>
<body bgcolor="white"><h2>Looking for DNA motifs Gibbs sampler (SeSiMCMC) result.</h2>
<p>Input FastA file name: 588780_0_03_MAR_22_38_10_extlz_sesimcmc.fasta</p>
<!-- $588780_0_03_MAR_22_38_10_extlz_sesimcmc.fasta -->
The best information content for different site lengthes:<br><br>
<table border=1><tr><td>Length</td><td><G></td><td></td></tr><tr><td>6</td><td>-0.00284506</td><td></td></tr>
<tr><td>7</td><td>0.0227135</td><td></td></tr>
<tr><td>8</td><td>0.0420698</td><td></td></tr>
<tr><td>9</td><td>0.0443861</td><td></td></tr>
<tr><td>10</td><td>0.0472596</td><td>***</td></tr>
<table><br>
The sampler was looking for  sites on both strands of 8 sequences of length from 213 through 237.<br><hr>
The site length found is 10.<br>
The sampler has tested the lengths in [6,10].<br>
The motif is formed by 8 motif sites.<br>
<br>All the positions are 0-based.<br><hr>
<table>
<caption><i><font size=+1>The description of sites</font></i></caption><tr><td>Gene Id</td><td>Position</td><td>Length</td><td>Score</td><td>Direction</td><td>Site in sequence</td><td width=10></td><td>Site-as-in-motif</td></tr>
<tr><td> 1</td><td align=center>102</td><td align=center>10</td><td align=center>10.68</td><td align=center>&gt</td><td><tt><font color=grey></font>CGCAGGACCt<font color=grey></font></tt></td><td></td><td><tt><font color=grey></font>CGCAGGACCt<font color=grey></font></tt></td></tr>
<tr><td> 2</td><td align=center>105</td><td align=center>10</td><td align=center>11.82</td><td align=center>&gt</td><td><tt><font color=grey></font>GCCAGGACCt<font color=grey></font></tt></td><td></td><td><tt><font color=grey></font>GCCAGGACCt<font color=grey></font></tt></td></tr>
<tr><td> 3</td><td align=center>107</td><td align=center>10</td><td align=center>8.938</td><td align=center>&lt</td><td><tt><font color=grey></font>gGGTCCTGCT<font color=grey></font></tt></td><td></td><td><tt><font color=grey></font>AGCAGGACCc<font color=grey></font></tt></td></tr>
<tr><td> 4</td><td align=center>109</td><td align=center>10</td><td align=center>5.952</td><td align=center>&lt</td><td><tt><font color=grey></font>tTATCCTTAT<font color=grey></font></tt></td><td></td><td><tt><font color=grey></font>ATAAGGATAa<font color=grey></font></tt></td></tr>
<tr><td> 5</td><td align=center>105</td><td align=center>10</td><td align=center>11.82</td><td align=center>&gt</td><td><tt><font color=grey></font>GCCAGGACCt<font color=grey></font></tt></td><td></td><td><tt><font color=grey></font>GCCAGGACCt<font color=grey></font></tt></td></tr>
<tr><td> 6</td><td align=center>106</td><td align=center>10</td><td align=center>10.58</td><td align=center>&gt</td><td><tt><font color=grey></font>GCAAGGACAt<font color=grey></font></tt></td><td></td><td><tt><font color=grey></font>GCAAGGACAt<font color=grey></font></tt></td></tr>
<tr><td> 7</td><td align=center>104</td><td align=center>10</td><td align=center>7.78</td><td align=center>&gt</td><td><tt><font color=grey></font>GCAGGGATAt<font color=grey></font></tt></td><td></td><td><tt><font color=grey></font>GCAGGGATAt<font color=grey></font></tt></td></tr>
<tr><td> 8</td><td align=center>102</td><td align=center>10</td><td align=center>8.316</td><td align=center>&gt</td><td><tt><font color=grey></font>CGCAGGATCg<font color=grey></font></tt></td><td></td><td><tt><font color=grey></font>CGCAGGATCg<font color=grey></font></tt></td></tr>
</table><br>
<i><font size=+1>The list of sites</font></i><br>
<pre>
 1   CGCAGGACCt
 2   GCCAGGACCt
 3   AGCAGGACCc
 4   ATAAGGATAa
 5   GCCAGGACCt
 6   GCAAGGACAt
 7   GCAGGGATAt
 8   CGCAGGATCg
</pre>
<br>
<!-- The configuration parameters:
mode=two_strands
symmetry=no
motif_absence_prior=0
adjust_motif_length=yes
spaced_motif=yes
common_background=no
adaptive_pseudocouts=yes
motif_length=10
maximal_motif_length=10
minimal_motif_length=6
retrieve_other_sites=no
retrieve_other_sites_threshold=0
trim_edges=no
dumb_trim_edges=no
caps=all
local_step_cycles_between_adjustments=0
adjustments_during_annealing=yes
annealing_criterion=information
cycles_with_minor_change_to_say_cold=5
minor_change_level=0.1
annealings_number_maximum_is_to_be_global_for=10
steps_number_maximum_is_to_be_global_for=0
slow_optimisation=yes
chain_fails_after=3
chains_to_try=3
cycles_per_annealing_attempt=0
annealing_attempts=5
random_seed_1=1248312
random_seed_2=7436111
background_A=0.25
background_T=0.25
background_G=0.25
background_C=0.25
-->
</body></html>
